pkg/blob/controllerserver.go

```diff
@@ -435,19 +442,19 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		setKeyValueInMap(parameters, containerNameField, validContainerName)
 	}
 
+	klog.V(2).Infof("begin to create container(%s) on account(%s) type(%s) subsID(%s) rg(%s) location(%s) size(%d)", validContainerName, accountName, storageAccountType, subsID, resourceGroup, location, requestGiB)
+	if err := d.CreateBlobContainer(ctx, subsID, resourceGroup, accountName, validContainerName, secrets); err != nil {
```

Comment:
azcopy would create container before volume clone, so let's revert this

Reply:
My intent here was for `CreateBlobContainer()` to configure `Access`, `PublicAccess`, `Metadata` uniformly. I see metadata on azcopy but not public access.

Reply:
ah, i see that actually an account setting not container setting i will make the change with metadata includes

Reply:
I have slight reservations still about not creating the blob container first with `CreateBlobContainer()` because azcopy can't default the blob container to private, so there's a slight difference in behavior.

However, since the default for the storage container is private this should only change the outcome when using a static storage account with public access turned on or when explicitly requested.

I left it as a separate commit so that you can decide whether that's a concern and either revert or squash.

Reply:
Ah, nope, the sanity check test threw this `failed to parse user input due to error: content-type, content-encoding, content-language, content-disposition, cache-control, or metadata is not supported while copying from service to service\n",`. I'll revert the change.